### ISSUE TITLE
Using static is not handled correctly for nested types

### ISSUE THREAD

**Issue by vladsch** [bridgedotnet/Bridge#3197]
A description of the issue.

### Steps To Reproduce

https://deck.net/838aa7e057a06400fb8d0cabb32c43d7

```csharp
using static Demo.pixi_js;

public class Program
{
    public static void Main()
    {
        var bunny = PIXI.Sprite.fromImage("required/assets/basics/bunny.png");
    }
}

namespace Demo
{
    public static class pixi_js
    {
        public static class PIXI
        {
             public class Sprite
             {
                  public static Sprite fromImage(string url)
                  {
                      return new Sprite();
                  }
             }
        }
    }
}
```

### Expected Result

```js
var bunny = Demo.pixi_js.PIXI.Sprite.fromImage("required/assets/basics/bunny.png");
```

### Actual Result

```js
[UnknownIdentifierResolveResult PIXI].sprite.fromImage("required/assets/basics/bunny.png");
```

**Comment by fabriciomurta** [comment]
The associated pull request has just been merged in. The fix will be available to Bridge 16.4.0.

**Comment by AndreyChechel** [comment]
@vladsch, I was able to reproduce that issue in Phaser.js demo project. The steps are:
1. Clone [Retyped.Demos](https://github.com/Retyped/Demos) repo
1. Open **PhaserDemo** solution
1. Edit [GameState1.cs (line 23)](https://github.com/Retyped/Demos/blob/4854ea2464f8de3993fd729621cbcfb20eb735d4/PhaserDemo/PhaserDemo/Games/GameState1.cs#L23) like it's shown below 

```cs
// works:
game.physics.enable(image, Physics.ARCADE);

// fails:
game.physics.enable(image, Phaser.Physics.ARCADE);
```

The generated output:
```js
// works:
this.game.physics.enable(image, Phaser.Physics.ARCADE);

// fails:
this.game.physics.enable(image, [UnknownIdentifierResolveResult Phaser].physics.aRCADE);
```

Do you think it'd be better to throw a compilation exception in such cases?

**Comment by vladsch** [comment]
Test case for new buggy scenario
https://deck.net/2d1f520a79e8075c8cfbb81af59ddd01

**Comment by fabriciomurta** [comment]
The test case was addressed via the recent pull request and was just merged in. This second case fix will be available to the next bridge version (currently 16.5.1).
The initial issue was fixed to Bridge 16.4.0.
